elide-integration-tests/src/test/java/com/yahoo/elide/async/integration/tests/AsyncIT.java

```diff
@@ -233,7 +257,9 @@ public void graphQLRequestTests() throws InterruptedException {
                                         attributes(
                                                 attr("query", "{\"query\":\"{ book { edges { node { id title } } } }\",\"variables\":null}"),
                                                 attr("queryType", "GRAPHQL_V1_0"),
-                                                attr("status", "QUEUED")
+                                                attr("status", "QUEUED"),
+                                                attr("requestId", "1001"),
+                                                attr("asyncAfterSeconds", "10")
```

Comment:
This doesn't match the prior test structure - where async after for 10 seconds does not require a subsequent for loop to test again and again.  Also - where is the test for 0 second async after.  Also, for 0 seconds, we should test the immediate response body.